torchreid/models/efficient_net_pytcv.py

```diff
@@ -355,7 +355,7 @@ def __init__(self,
             self.output.add_module("asl", AngleSimpleLinear(
                 in_features=final_block_channels,
                 out_features=self.num_classes))
-        elif 'softmax' == self.loss or 'asl' == self.loss:
+        elif 'softmax' in self.loss or 'asl' in self.loss:
```

Comment:
Based on this code, I'm not sure if self.loss is an iterable object. 
Even the if statements before the elif statements still use '=='. I wonder if this is a bug fix.

Reply:
self.loss is incoming to this line as string. ex) 'softmax', 'softmax,asl'

But by the modification from #220 , current code allows and parse only when self.loss is exactly same string. This update does not handle multihead classifciation : it uses self.loss as 'softmax,asl'.

So, we need this modification.

Reply:
But I'm also wondering whether the above if line also should be changed from == to 'in'. 
Because this can misunderstanding - if we change above == to 'in', when we run multihead task and define self.loss as 'am_binary', the experiments will runs fine but 'am_binary' will not be actually adopted without any error. 
This is the reason why I didn't change above == to 'in'.

Reply:
That is understood but does not seem appropriate in code.
How about this?
```
elif self.loss in ['softmax', 'asl', 'softmax.asl']:
```

Reply:
I think that's fine. Do you think also changing above if line is appropriate?

Reply:
I think it doesn't matter if you change to the same convention with the 'if' statement and 'elif' statement.

Reply:
Reflected. Thank you for your suggestion.

Reply:
Is that comma? Then one more suggestion could be a explicit tokenization by ',' then mathcing exact name with '=='.

Reply:
Yes, that's comma. I also found multihead engine splits loss : https://github.com/openvinotoolkit/deep-object-reid/blob/ote/torchreid/engine/image/multihead.py#L48

Reply:
Revised like below code:
`self.loss = self.loss.split(',') if ',' in self.loss else self.loss`

Reply:
String returns a list if you use split, so why don't you think of 'self.loss' as a list and match the rest of the inspection code?
`self.loss = self.loss.split(',')` returns the list without the need for an if & else statement.

Reply:
Updated. Thank you.